### ISSUE TITLE
 Does anybody know about a problem  with courseplay and spreading slurry?

### ISSUE THREAD

**Issue by GerryBee** [Courseplay/courseplay#2785]
I'm using the Horsch slurry tanker from the Horsch DLC. Somehow at the moment the helper starts working I get a red message saying:"helper N stopped working unexpectedly, tanker empty". But the tanker is full and courseplay just loaded it full of slurry , now the helper starts riding the course but is spraying nothing and, after a while the machine running the tanker slows down until it stops. Meanwhile the message has multiplied a couple of times in my screen.
First time it happened was on Lone Oak Map whilst playing it with seasons and realistic installed.
I started testing on standard map and now seem to have the same problem with all original barrels (no mods until issue is solved).
If anybody could help me out please...I've also checked out Github and Seasons page and I did have some mods installed that were in the "bad" list, so I removed them and started a new game in an new slot....but again no slurry spreading with courseplay (latest official version).
I am a big fan of courseplay and I have been using courseplay intesively since fs2013, so I m very familiar with it. I think it's a mod conflict.

**Comment by pops64** [comment]
could we get a video? Are accidentally hitting the 'H' key after CP is running? Log with debug channel 17 active, have you tried using digesate instead of slurry

**Comment by GerryBee** [comment]
Thank you for the quick reply. The "h" for the Helper, no, for sure. I'm going to try the digesate option now, but i it's getting late here , tomorrow i have to work early. I 'll let you know and /or send you the log tomorrow.

**Comment by GerryBee** [comment]
Hey. it's complicated, but I've managed the Horsch barrel to run again, on Lone Oak Farmrunning it full realistic with seasons. First I tested with the standard Joskin barrel and it worked perfectly without the red message popping up on my screen, after I took on the Horsh barrel again and managed to get it to spray.  However courseplay didn't start the spraying, therefore I had to stop courseplay, put cruise control on lower the hoses and then click courseplay on and it took it over whilst spraying, Since that moment it worked, on this map (but the failure notice does not disappear!) . When I went back to standard map, without seasons, the barrel does not spray at all, also with digesate on. Now before I send you some footage taken, I have another thing that might cause this issue. Before this all started I had GameExtension installed, and off course I've removed it as iI got into the matter. I discovered some remnants of this mod in the game folder:- GameExtension_Client and: -GameExtension_Server. Could it be that hey are the cause? After I took them out and started testing on the standard map courseplay started malfunctioning: vehicles start the course but then it says they are in traffic while there is none.(even the option is turned off). After I opened the log and ther was this warning:
WARNING: 
undefined input in gearboxMogli: 
C:/Users/Barbara/Documents/My Games/FarmingSimulator2017/mods/FS17_SafetyWeight/SafetyWeight.i3d (648.65) ms
C:/Users/Barbara/Documents/My Games/FarmingSimulator2017/mods/FS17_horschAgrovationVehicles/vehicles/trailers/horschSlurryTank21000.i3d (2173.40) ms
dataS2/character/pedestrians/woman01.i3d (7.52) ms
C:/Users/Barbara/Documents/My Games/FarmingSimulator2017/mods/FS17_horschAgrovationVehicles/vehicles/tools/horschDragHose36.i3d (342.53) ms
C:/Program Files (x86)/Steam/steamapps/common/Farming Simulator 17/pdlc/platinumEdition/character/pedestrians/woman10.i3d (724.71) ms
dataS2/character/pedestrians/man02.i3d (50.20) ms
dataS2/character/pedestrians/woman03.i3d (144.03) ms
C:/Program Files (x86)/Steam/steamapps/common/Farming Simulator 17/pdlc/platinumEdition/character/pedestrians/man10.i3d (461.32) ms
dataS2/character/pedestrians/man01.i3d (18.46) ms
dataS2/character/pedestrians/man06.i3d (20.72) ms
C:/Program Files (x86)/Steam/steamapps/common/Farming Simulator 17/pdlc/platinumEdition/character/pedestrians/man08.i3d (473.05) ms
dataS2/character/pedestrians/woman05.i3d (14.37) ms
dataS2/character/pedestrians/man05.i3d (16.92) ms

When I put the 2 xml-files back in folder the course works fine only still no spreading...
So I wander, should I completely reinstall the game?

**Comment by pops64** [comment]
Do you have it installed on Steam? If so, verify the game cache has you might have a corrupted file. Lets just stick to vanilla map and vanilla slurry tankers for testing to eliminate the possibly of a mod conflict or dlc conflict. I know I was getting some weird behavior like this on Lone Oak during testing of another bug(multiple start stops causes the sower not to turn on but all actions appeared to still work, I also know there is an error on start up of Lone Oak)

**Comment by GerryBee** [comment]
Well Pops64,
It seems like I've answered my own question, it was definetly the remnants of the GameExtension mod. Yesterday I decided to totally reinstall (and yes I' m on Steam) and now it works fine!  So in the end it was of course my mistake, since I had fine timescaleadjust-mod, faster night mod, Bank of Hagensted all installed at the moment I installed Lone Oak Farm together with the seasons mod. And when I discovered those mods do not cooperate, I immediately removed them. The remnants however force you to totally reinstall the game you just cannot remove these xml's from the gamefolder since that leads to a complete malfunctioning courseplay. Anyway, thank you again for your quick reply and help. Now I'm gonna play this map and if there are any issues as you mentioned I will let you know.
Greetz...GerryBee.
